apps/api/v2/src/ee/bookings/2024-08-13/bookings.repository.ts

```diff
@@ -169,4 +169,33 @@ export class BookingsRepository_2024_08_13 {
       },
     });
   }
+
+  async getAcceptedUserBookingForTimeRange(userId: number, startTime: Date, endTime: Date) {
+    return this.dbRead.prisma.booking.findFirst({
+      where: {
+        userId,
+        status: "ACCEPTED",
+        startTime: {
+          gte: startTime,
+          lt: endTime,
```

Comment:
Personally, would set to `lte` since we have `gte` aka we include the start - i think then it makes sense to include the end too.